### ISSUE TITLE
kcm-wacomtablet not working

### ISSUE THREAD

**Issue by poscat0x04** [NixOS/nixpkgs#104384]
**Describe the bug**
The KDE Wacom configuration module is not working, showing "The shared library was not found":

**To Reproduce**
1. use KDE
2. set `services.xserver.wacom.enable = true`
3. open KDE system settings

**Expected behavior**
The configuration module should work

**Screenshots**
![image](https://user-images.githubusercontent.com/53291983/99806818-82c64080-2b79-11eb-907e-6402cc599604.png)

**Additional context**

**Notify maintainers**

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.77, NixOS, 21.03.20201115.069f183 (Okapi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.4pre20201102_550e11f`
 - channels(root): `""`
 - nixpkgs: `/nix/store/7nkxq994g75sbygb0whcjqcpwk9cyljh-source`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: services.xserver.wacom
# a list of nixos modules affected by the problem
module:
```

**Comment by cirno-999** [comment]
can confirm
![image](https://user-images.githubusercontent.com/73712874/99909124-f76abd80-2ce6-11eb-89c2-e28c68da6e2a.png)

 - system: `"x86_64-linux"`
 - host os: `Linux 5.8.16, NixOS, 20.09.1469.13d0c311e3a (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 3.0pre20201020_e0ca98c`
 - channels(root): `"nixos-20.09.1469.13d0c311e3a"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by poscat0x04** [comment]
not sure if this is related to #102253

**Comment by petabyteboy** [comment]
This is still be a problem on 20.09

**Comment by schmittlauch** [comment]
According to strace, the missing library file is `libkcm_wacomtablet.so`.

**Comment by zachlatta** [comment]
Running into same problem now on 20.09.

**Comment by Thra11** [comment]
Just to clarify, does #104660 just need backporting to 20.09, or are people still experiencing issues on master after #104660 was merged?

**Comment by zachlatta** [comment]
I think it just needs backporting, but I'm not yet familiar enough with NixOS to say for sure. I just upgraded my system to unstable and `kcm-wacomtablet` is working as expected (which I'm assuming is because of #104660).

![image](https://user-images.githubusercontent.com/992248/105086286-d7e1de00-5a66-11eb-9926-6af79e7da22e.png)

**Comment by Thra11** [comment]
There was a PR to backport it to 20.09, but nobody reviewed it and it was closed when 20.09 went end-of-life, so I'm assuming everybody using kcm-wacomtablet is now happily running 21.05 or unstable and we can close this.

**Comment by schmittlauch** [comment]
Yes, I can confirm it is working again in 21.05.
